### DISCUSSION TITLE
Unable to restart a failed workflow

### DISCUSSION THREAD

**Discussion by lmilbaum** [python-gitlab/python-gitlab#2359] [Q&A]
I'm guessing the capability exists and it is just a matter of permissions

**Comment by nejch** [comment]
Yes, it seems that permissions aren't granular like that and you need write access to the repository:

https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role

I think we can also do that after a while as part of the checklist here when we get back to it (probably in a different form) https://github.com/python-gitlab/python-gitlab/pull/2224

**Reply by lmilbaum** [reply]
I am checking some other options. A PR is on its way. :-)
